moviebot/controller/controller.py

```diff
@@ -45,3 +48,46 @@ def restart(self, utterance: UserUtterance) -> bool:
             utterance: User utterance.
         """
         return utterance.get_text() == RESTART
+
+    def get_cursor(self) -> sqlite3.Cursor:
+        """Returns SQL cursor."""
+        conn = sqlite3.connect(self.configuration["DATA"]["db_path"])
+        c = conn.cursor()
+        return c
+
+    def load_user_data(self, path: str, user_id: str) -> Dict[str, Any]:
+        """Loads movie choices (accept/reject) for a user from conversation
+        history.
+
+        Args:
+            path: Path to conversation histories.
+            user_id: User id.
+
+        Returns:
+            Object with previous movie choices.
+        """
+        user_history_path = f"{path}user_{user_id}.json"
+        loaded_data = {}
+        if os.path.isfile(user_history_path):
+            with open(user_history_path) as json_file:
+                data = json.load(json_file)
+                for conversation in data:
+                    for movie in conversation["Context"]:
+                        loaded_data[movie] = conversation["Context"][movie]
+        return loaded_data
+
+    def delete_history(self, path: str, user_id: str) -> bool:
+        """Deletes stored conversation history for user.
+
+        Args:
+            path: Path to conversation histories.
+            user_id: User id.
+
+        Returns:
+            A boolean indicating whether or not history was deleted.
+        """
+        user_history_path = f"{path}user_{user_id}.json"
```

Comment:
See above. 
Eiter make a global template for `user_history_path`, or return it in a separate method. We do not want to have to maintain the template in several places (here and in `load_user_data`) if we decide to change the folder structure.

Reply:
I create a method `get_user_history_path` to return the path to conversation history.